docs/fragments/lib/android.md

```diff
@@ -1,15 +1,17 @@
-## Amplify Android (Preview)
+## Amplify Android
 
-This guide shows how to build an app using our Amplify Libraries for Android (Preview) and the Amplify CLI toolchain. To use the existing AWS Mobile SDK for Android, refer to the [AWS Mobile SDK for Android guide](~/sdk/sdk.md).
+This guide shows how to build an app using our Amplify Libraries for Android and the Amplify CLI toolchain.
 
 <docs-internal-link-button href="~/lib/project-setup/prereq.md">
   <span slot="text">Get Started 🚀</span>
 </docs-internal-link-button>
 
+## AWS Mobile SDK for Android
+We highly recommend that you adopt Amplify, but if you are still using our low level SDK for Android, you can find the documentation here: [AWS Mobile SDK for Android](~/sdk/sdk.md).
+
 ## FAQs
```

Comment:
We should move this FAQ to the landing page for the Mobile SDK (~/sdk/sdk.md). It isn't relevant to a new developer starting with Amplify Android.

Reply:
Sure, I can update this.. do you have any recommendations on how we should phrase this?  For example, maybe a question might be:

Q: Should I use Amplify or AWS Mobile SDK?
A: We highly recommend the adoption of Amplify over the AWS SDK.  Amplify provides modern, high level programmable interfaces that hide complexity of the low level interactions with service level clients.

Reply:
hmm given the other content on this page, i think it's good enough to answer the question with simply: "We highly recommend the adoption of Amplify over the AWS SDK."